connector/src/test/scala/com/google/cloud/spark/bigquery/it/SparkBigQueryEndToEndITSuite.scala

```diff
@@ -157,6 +157,20 @@ class SparkBigQueryEndToEndITSuite extends FunSuite
       assert(row(1).isInstanceOf[String])
     }
 
+    test("cache data frame in DataSource %s. Data Format %s".format(dataSourceFormat, dataFormat)) {
+      val allTypesTable = readAllTypesTable("bigquery")
+      writeToBigQuery(allTypesTable, SaveMode.Overwrite, "avro")
+
+      val df = spark.read.format("bigquery")
+        .option("dataset", testDataset)
+        .option("table", testTable)
+        .option("readDataFormat", "arrow")
+        .load().cache()
+
+      assert(df.head() == allTypesTable.head())
```

Comment:
Can you please duplicate this line? the first time the data will be read from BigQuery, the second time from the cache